### PR TITLE
[TS-000] Collection-render helper

### PR DESCRIPTION
**Pull request by im-kulikov**

### CODE REVIEW

**Review by kpeo — DISMISSED** [review]

File: helpers/collection/collection.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		return errors.NewError(http.StatusBadRequest, err.Error())
+	}
+
+	if response.Total, err = opts.Query.Count(); err != nil {
+		ctx.Logger().Error(err.Error())
+		return errors.NewError(http.StatusInternalServerError, err.Error())
+	}
+
+	// Pagination:
+	opts.Query.Limit(pager.Limit)
```
Comment (jenchik) [inline helpers/collection/collection.go:82]: mb ```opts.Filter.Pagination(opts.Query)```

File: helpers/collection/collection.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	opts.Query.Offset(pager.Offset)
+
+	if response.Items, err = opts.Fetcher(opts.Query); err != nil {
+		ctx.Logger().Error(err.Error())
+		return errors.NewError(http.StatusInternalServerError, err.Error())
+	}
+
```
Comment (jenchik) [inline helpers/collection/collection.go]: response.Offset = ?

Comment (im-kulikov) [reply helpers/collection/collection.go]: oh.. yeah

File: helpers/collection/collection.go (hunk 1 above)
Comment (im-kulikov) [reply helpers/collection/collection.go:82]: nope... it bad case..

**Review by jenchik — DISMISSED** [review]

**Review by jenchik — APPROVED** [review]

**Review by kpeo — APPROVED** [review]

**Pull request merged**